services/web/server/src/simcore_service_webserver/studies_dispatcher/_projects.py

```diff
@@ -103,7 +103,8 @@ def create_viewer_project_model(
         inputs={},
         inputNodes=[],
         outputs={
-            file_picker_output_id: DownloadLink(downloadLink=str(download_link), label=None)
+            # TODO: check with odeimaiz what is the meaning and relevance of label
+            file_picker_output_id: DownloadLink(downloadLink=str(download_link), label="")
```

Comment:
and can't you create it without label if it's empty?

Reply:
Yes. it will add a None in the data instance, then nothing in the database (because ``json(exclude_none=True, by_alias=True)`` and then back to None when loaded in ``ProjectsAtDB``... BUT Odei uses en empty string himself in the front-end and I am not sure this will have an effect... therefore I keep it that way for the moemnt